gprofiler/main.py

```diff
@@ -146,6 +146,7 @@ def _snapshot(self):
                 logger.error(f"Error occurred sending profile to server: {e}")
             except RequestException:
                 logger.exception("Error occurred sending profile to server")
+            logger.info("Successfully uploaded profiling data to the server")
```

Comment:
I think this should go to the `else` clause? Here you'll log "successfully" also after failures.

Reply:
So either in the `else` clause of the `try`, or after the `submit_profile(...)` call